.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 phpunit.xml
 vendor
 .php_cs.cache
+.idea
```

Comment:
This should be in your global gitignore instead. Other we'll end up with all different OS/IDE-specific files.